CHANGES

```diff
@@ -93,6 +93,18 @@ Changes with Apache 2.4.47
      but others reject.
      [Stefan Eissing, @tlhackque, Andreas Ulm]
 
+  *) mod_dav_fs: Improve logging output when failing to open files for
+     writing.  PR 64413.  [Bingyu Shen <ahshenbingyu gmail.com>]
+
+  *) mod_http2: Fixed a race condition that could lead to streams being
+     aborted (RST to the client), although a response had been produced.
+     [Stefan Eissing]
+
+  *) mod_lua: Add support to Lua 5.4 [Joe Orton, [Giovanni Bechis, Ruediger Pluem]
+
+  *) MPM event/worker: Fix possible crash in child process on early signal
+     delivery.  PR 64533.  [Ruediger Pluem]
+
```

Comment:
Best thing is to leave all change entries out that do not belong to this backport. In an ideal world they would all get backported by files in `change-entries` and would be added later (latest with the roll script) by executing `make update-changes`.

Reply:
Agreed. With the recent back and forth about this, I was not sure what the PR should carry.

Reply:
Can you please remove the `CHANGES` entries and put the changes for this backport in a file below `change-entries`? Once done I would vote +1 such that we could get this in 2.4.48.

Reply:
Did the changes. Agree with you that this is the preferred way now.